### DISCUSSION TITLE
[2021.09.01] 주식 가격

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#91] [Algorithm]
[주식 가격](https://programmers.co.kr/learn/courses/30/lessons/42584?language=java)

**Comment by hyk339** [comment]
[주식가격 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_42584.java)

**Reply by hyk339** [reply]
문제 이해가 어려워 문제부분만 다른분들의 풀이를 참고했습니다.

**Comment by pyunji** [comment]
[주식 가격 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42584.java)

**Comment by leechang4** [comment]
어렵습니다.. 내일 설명해 주시면 감사하겠습니다..

**Comment by chaeyeon4u** [comment]
[주식 가격 solved](https://programmers.co.kr/learn/courses/30/lessons/42584?language=java)
다른 사람 풀이를 많이 참고해서 오늘 안보고 다시 풀어보겠습니다..!

**Comment by leechang4** [comment]
[주식 가격 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR_42584.java)
